### DISCUSSION TITLE
希望可以为动态增加一个“查看详情”功能

### DISCUSSION THREAD

**Discussion by 1357310795** [the1812/Bilibili-Evolved#2093] [功能建议 / Ideas]
就是增加一个进入单独动态页的按钮
效果如图：
<img width="1076" alt="屏幕截图 2021-07-27 205802" src="https://user-images.githubusercontent.com/35869804/127157653-bccbe86b-894c-4427-b55a-17b1e825faa0.png">

有时候希望单独分享一个动态时会用到。
求翻牌！谢谢大佬了😁

**Comment by the1812** [comment]
点那个时间就是

**Reply by the1812** [reply]
另外也有复制动态链接的功能

**Comment by Geeyun-JY3** [comment]
要么别开 `禁止跳转动态详情`功能，直接点击动态文字进入动态详情；要么就通过点击时间戳进入动态详情。

点击时间戳进入详情是一种通用做法。通常都能通过时间戳获得指向该内容的链接，只是国内很多网站都把这给阉掉了。
